paasta_tools/kubernetes_tools.py

```diff
@@ -2471,7 +2471,9 @@ def get_pod_template_spec(
         if self.should_use_metrics_provider(METRICS_PROVIDER_PISCINA):
             labels["paasta.yelp.com/scrape_piscina_prometheus"] = "true"
 
-        if self.should_use_metrics_provider(METRICS_PROVIDER_GUNICORN):
+        if self.should_use_metrics_provider(
+            METRICS_PROVIDER_GUNICORN
+        ) or self.should_use_metrics_provider(METRICS_PROVIDER_WORKER_LOAD):
```

Comment:
i'd probably add a comment here since we'll probably have a gunicorn/uwsgi mix for quite some time and a `paasta.yelp.com/scrape_gunicorn_prometheus` label on a uwsgi pod might lightly confuse folks in the future :p

Reply:
(i think ideally we'd do a little bit of a dance here and swap this to a more generic label - but imo a comment is fine for now)